dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/constants/DataSourceConstants.java

```diff
@@ -28,7 +28,7 @@ public class DataSourceConstants {
     public static final String COM_MYSQL_CJ_JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
     public static final String COM_MYSQL_JDBC_DRIVER = "com.mysql.jdbc.Driver";
     public static final String ORG_APACHE_HIVE_JDBC_HIVE_DRIVER = "org.apache.hive.jdbc.HiveDriver";
-    public static final String COM_CLICKHOUSE_JDBC_DRIVER = "ru.yandex.clickhouse.ClickHouseDriver";
```

Comment:
are u sure using com.xxx prefixed driver can run clickhouse job normally?

Reply:
yes, `ru.yandex.clickhouse` is deprecated, please refers to https://github.com/ClickHouse/clickhouse-jdbc#clickhouse-java-client--jdbc-driver